bot.py

```diff
@@ -6,8 +6,12 @@
 
 
 class Bot:
-    def __init__(self, conn, iden, name):
-        self.__id = iden
+    """
+    Class is a abstraction of bot.
```

Comment:
If you want it abstract then make it absctract 😉 
https://stackoverflow.com/questions/13646245/is-it-possible-to-make-abstract-classes-in-python

Reply:
When I wrote these lines, I meant `abstraction` in another sense of the word :P